content/events/openSciencePractices2025.json

```diff
@@ -10,8 +10,8 @@
     "venueAddress": "",
     "venueMapsLink": "",
     "website": "https://iosp.io/schedule",
-    "startDate": "2025-02-22T00:00:00.000Z",
-    "endDate": "2025-02-23T00:00:00.000Z",
+    "startDate": "2025-02-24T00:00:00.000Z",
```

Comment:
@WinsCodes Can you please change these dates to begin on 2/23/2025 and end on 2/25/2025?